### PR TITLE
Polish SSO-SAML

### LINKED ISSUE #2433: Some SSO-SAML improvements
- [ ] All password related fields should be invisble for users with a saml_id
  - Password field in user details (edit)
  - Password reset in "my account"
  - Set password as superadmin
  - Bulk update for password (three-dot-menu - "password reset to the default ones"; "Generate new passwords")
- [ ] Show saml_id in account/participant list and detail pages
- [ ] Simplify login page if SAML is enabled
  - The SAML login button should be more present (first element)
  - The login with username an password should be wrapped in a collapsible box (collapsed by default)
- [ ] The SMAL config fields should get more descriptive labels and some helptext.

### LINKED ISSUE #2439: The SAML config fields should get more descriptive labels and some helptext.
Some fields in the orga settings have labels like `SSO IDP`, which explain very little about what is expected.
They should have more verbose labels and help texts that explain the functionality.

The content of these help texts and labels have currently not been decided on.
What should they be @emanuelschuetze ?

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2433 
Closes #2439

### CODE REVIEW

File: client/src/app/infrastructure/utils/functions.ts [hunk 1]
```diff
@@ -312,8 +312,11 @@ export function objectToFormattedString(jsonOrObject: string | object): string {
     if (!jsonOrObject) {
         return undefined;
     }
-    let json =
-        jsonOrObject && typeof jsonOrObject !== `string` ? JSON.stringify(jsonOrObject) : (jsonOrObject as string);
+    let json = (
+        jsonOrObject && typeof jsonOrObject !== `string` ? JSON.stringify(jsonOrObject) : (jsonOrObject as string)
+    )
+        .split(/\n|\s/)
+        .join(``);
```
Comment (bastianjoel) [inline client/src/app/infrastructure/utils/functions.ts:319]: I don't think this is a good idea as this could also change content of properties.

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/infrastructure/utils/functions.ts (hunk 1 above)
Comment (luisa-beerboom) [reply client/src/app/infrastructure/utils/functions.ts:319]: This function is supposed to format object json into human-readable strings, taking out pre-existing strings and line-breaks before adding in the desirable ones is pretty necessary IMO.

I don't think the result of this is going to be saved anywhere where it wouldn't be fine to save the transformed value.

Or did you mean something else?

Comment (bastianjoel) [inline client/src/app/infrastructure/utils/functions.ts:319]:
```suggestion
    let json = (
        jsonOrObject && typeof jsonOrObject !== `string` ? JSON.stringify(jsonOrObject) : JSON.stringify(JSON.parse(jsonOrObject as string))
    );
```

**Review by bastianjoel — APPROVED** [review]

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
Password functions still available for SSO account:
* Password reset in "my account" (as SSO user)
* Change password from account detail (as superadmin)
* Bulk update for password (three-dot-menu - "Generate new password")

**Review by Elblinator — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]
code-review

**Pull request merged**